### PR TITLE
Added script for generating content key using Key Server

### PR DESCRIPTION
**Pull request by Heronyme**
Also modified readme.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -135,9 +135,12 @@ Makemedia system requirements:
 
 **The starting point for this scenario is an MP4 or MOV file containing both video and audio** - ensure that your video is in this format before continuing. You can download some free test content in this format from the [Blender Foundation](https://mango.blender.org/download/). The steps below will transform this input video into a format suitable for playback.
 
-Before you can do anything with the video, you will need to generate a content key. The sample project includes a command line application for this. Simply execute the following command in the project directory: *node GenerateKey.js*
+Before you can do anything with the video, you will need to generate a content key. The sample project includes two simple command line applications that allow generating either a totally random key or a random key that is based on your key seed and a randomly generated key ID:
```
Comment (sandersaares) [inline Readme.md]: I would instead focus on the difference of *where* you generate it. How it is generated seems far less important - what matters in my imagination is whether the generation happens on the user's PC or on a key server.

File: Readme.md [hunk 2]
```diff
@@ -135,9 +135,12 @@ Makemedia system requirements:
...
 
+* *GenerateKey.js* - generates a random content key. To use it, execute the following command in the project directory: *node GenerateKey.js*
```
Comment (sandersaares) [inline Readme.md]: Maybe it also makes sense to rename this to GenerateKeyOnLocalPc.js? This way it is more clear what the difference really is (that this one does not use a key server).

File: package.json [hunk 3]
```diff
@@ -7,10 +7,12 @@
   "license": "MIT",
   "repository": "https://github.com/Axinom/drm-quick-start",
   "dependencies": {
+    "commander": "^2.9.0",
```
Comment (sandersaares) [inline package.json]: I wonder if the new entries should be devDependencies instead? I always forget the exact logic - just something that should be double checked... maybe it does not even matter.

File: GenerateKeyUsingKeyServer.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env node
```
Comment (sandersaares) [inline GenerateKeyUsingKeyServer.js]: This file does not pass ESLint validation. 15 errors, 6 warnings.

File: Readme.md (hunk 1 above)
Comment (KristjanJ) [inline Readme.md]: Yes, clarify that in the 2nd case it's generated on the Key Server, but still mention that it's based on the customer's default Key Seed.

File: package.json (hunk 3 above)
Comment (Heronyme) [inline package.json]: I understand devDependencies are those, which shouldn't required for "normal" use. More like extra modules for executing tests, creating/combining new minified packages and such.

File: Readme.md (hunk 2 above)
Comment (Heronyme) [inline Readme.md]: I'd leave it like this atm. Simple script and name :) Will change description a bit.

File: GenerateKeyUsingKeyServer.js (hunk 4 above)
Comment (Heronyme) [inline GenerateKeyUsingKeyServer.js]: Thanks, fixed most. Left "200" magic number.

**Comment by sandersaares** [comment]
Looks good to me.

**Pull request merged**